modules/scip/9.2.0.bcr.1/patches/add_buildflags_h.patch

```diff
@@ -2,31 +2,62 @@ diff --git a/src/scip/buildflags.h b/src/scip/buildflags.h
 index c686abcd20..1ea6a4fbf4 100644
 --- a/src/scip/buildflags.h
 +++ b/src/scip/buildflags.h
-@@ -29,10 +29,44 @@
- 
- /*---+----1----+----2----+----3----+----4----+----5----+----6----+----7----+----8----+----9----+----0----+----1----+----2*/
- 
+@@ -0,0 +1,94 @@
++/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
++/*                                                                           */
++/*                  This file is part of the program and library             */
++/*         SCIP --- Solving Constraint Integer Programs                      */
++/*                                                                           */
++/*  Copyright (c) 2002-2024 Zuse Institute Berlin (ZIB)                      */
++/*                                                                           */
++/*  Licensed under the Apache License, Version 2.0 (the "License");          */
++/*  you may not use this file except in compliance with the License.         */
++/*  You may obtain a copy of the License at                                  */
++/*                                                                           */
++/*      http://www.apache.org/licenses/LICENSE-2.0                           */
++/*                                                                           */
++/*  Unless required by applicable law or agreed to in writing, software      */
++/*  distributed under the License is distributed on an "AS IS" BASIS,        */
++/*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. */
++/*  See the License for the specific language governing permissions and      */
++/*  limitations under the License.                                           */
++/*                                                                           */
++/*  You should have received a copy of the Apache-2.0 license                */
++/*  along with SCIP; see the file LICENSE. If not visit scipopt.org.         */
++/*                                                                           */
++/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
++
++/**@file   buildflags.h
++ * @brief  build flags methods
++ * @author Felipe Serrano
++ */
++
++/*---+----1----+----2----+----3----+----4----+----5----+----6----+----7----+----8----+----9----+----0----+----1----+----2*/
++
 +
 +#if defined(__GNUC__) && defined(__linux__)
```

Comment:
Nit: You probably want to remove the GNU part of the if (Linux+Clang is still Linux)

Reply:
Done